go.mod

```diff
@@ -139,30 +139,6 @@ require (
 )
 
 replace (
-	cloud.google.com/go => cloud.google.com/go v0.75.0
-	github.com/go-openapi/spec => github.com/go-openapi/spec v0.19.6 // indirect
-	github.com/go-openapi/swag => github.com/go-openapi/swag v0.19.7 // indirect
-	github.com/gofrs/flock => github.com/gofrs/flock v0.7.1
-	github.com/google/uuid => github.com/google/uuid v1.1.4 // indirect
-	github.com/hashicorp/golang-lru => github.com/hashicorp/golang-lru v0.5.4 // indirect
-	github.com/imdario/mergo => github.com/imdario/mergo v0.3.11 // indirect
-	github.com/mrunalp/fileutils => github.com/mrunalp/fileutils v0.5.0
-	github.com/onsi/ginkgo => github.com/onsi/ginkgo v1.14.1 // indirect
-	github.com/onsi/gomega v1.10.3 => github.com/onsi/gomega v1.10.3 // indirect
-	github.com/spf13/cobra => github.com/spf13/cobra v1.4.0
-	github.com/spf13/pflag => github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify => github.com/stretchr/testify v1.7.0
-	go.uber.org/zap => go.uber.org/zap v1.17.0 // indirect
-	golang.org/x/lint => golang.org/x/lint v0.0.0-20201208152925-83fdc39ff7b5 // indirect
-	golang.org/x/oauth2 => golang.org/x/oauth2 v0.0.0-20211104180415-d3ed0bb246c8
-	golang.org/x/sync => golang.org/x/sync v0.0.0-20201207232520-09787c993a3a // indirect
-	google.golang.org/api => google.golang.org/api v0.63.0
-	google.golang.org/genproto => google.golang.org/genproto v0.0.0-20210111234610-22ae2b108f89 // indirect
-	google.golang.org/grpc => google.golang.org/grpc v1.34.0 // indirect
-	gopkg.in/check.v1 => gopkg.in/check.v1 v1.0.0-20200902074654-038fdea0a05b // indirect
-	gopkg.in/gcfg.v1 => gopkg.in/gcfg.v1 v1.2.3
-	gopkg.in/warnings.v0 => gopkg.in/warnings.v0 v0.1.2
```

Comment:
@jprzychodzen I think these are safe to remove, at least I couldn't find a reason, it may be a legacy thing to deal with vendoring, right?